its/tests/src/test/java/its/SonarQubeDeveloperEditionTests.java

```diff
@@ -181,7 +181,7 @@ static void start() {
     backend = new SonarLintBackendImpl(newDummySonarLintClient());
     try {
       backend.initialize(
-        new InitializeParams(IT_CLIENT_INFO, new FeatureFlagsDto(false, true, false, false, false, true), sonarUserHome.resolve("storage"), sonarUserHome.resolve("workDir"),
+        new InitializeParams(IT_CLIENT_INFO, new FeatureFlagsDto(false, true, false, false, false, true, false), sonarUserHome.resolve("storage"), sonarUserHome.resolve("workDir"),
```

Comment:
Do we have at least one IT to validate this feature? Or are we maybe waiting for changes on the DBD plugin side?

Reply:
Yup, writing an IT for this feature would be premature right now.

Reply:
Maybe create a SLCORE ticket to not forget

Reply:
Created [SLCORE-660](https://sonarsource.atlassian.net/browse/SLCORE-660), thanks!